docs/draftOpenAPIChallenge.md

```diff
@@ -23,15 +23,18 @@ npm install -g swagger-inline
 
 2. Add OpenAPI endpoints inline comments to the application API endpoints
 
-The comments should be added to the sample app `server.js` file where the app API endpoints are defined. The comments should be added above the:
-- [visitors POST API](https://github.ibm.com/developer-first-guild/get-started-openapi/blob/master/server.js#L38)
+The comments should be added to the sample app `server.js` file where the app API endpoints are defined. The [visitors POST API](https://github.ibm.com/developer-first-guild/get-started-openapi/blob/master/server.js#L38) comment has already being added for guidance.
```

Comment:
With the addition of the example for the /post - the line numbers increment by 22. I've updated the content in the Challenge on Influitive accordingly to 60, 88, 104 and 134 for the respective endpoints.

Reply:
Also in the challenge corrected phrase to "...comment has already been added for guidance."

Reply:
I started to add the updates from this file to Influitive, but until server.js is merged this won't work so holding off on that until the PR is merged.

Reply:
Good comments. I updated:
- The `server.js` line numbers
- Changed 'being' to 'been'

Good idea to hold back on updating the coder program, until the PR gets merged.